<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="30" passed="22" failed="5" skipped="3">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-03-04T21:11:04 IST" name="Default suite" finished-at="2022-03-04T21:11:17 IST" duration-ms="12693">
    <groups>
    </groups>
    <test started-at="2022-03-04T21:11:04 IST" name="Default test" finished-at="2022-03-04T21:11:17 IST" duration-ms="12693">
      <class name="com.epam.TestAutomation.API_Assignment.TestAPI">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:04 IST" name="setUp" finished-at="2022-03-04T21:11:04 IST" duration-ms="490" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testFetUserWithIdWithoutToken()[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:04 IST" name="testFetUserWithIdWithoutToken" finished-at="2022-03-04T21:11:07 IST" duration-ms="3043" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFetUserWithIdWithoutToken -->
        <test-method signature="testGetResponseWithInvalidToken(java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:07 IST" name="testGetResponseWithInvalidToken" data-provider="InvalidTokens" finished-at="2022-03-04T21:11:08 IST" duration-ms="334" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[c9c1347c-698a-ab93-9504-43726nhf70c4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetResponseWithInvalidToken -->
        <test-method signature="testGetResponseWithInvalidToken(java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:08 IST" name="testGetResponseWithInvalidToken" data-provider="InvalidTokens" finished-at="2022-03-04T21:11:08 IST" duration-ms="283" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[c9c1247c-698a-cd93-9504-43726adf70c4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetResponseWithInvalidToken -->
        <test-method signature="testGetResponseWithInvalidToken(java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:08 IST" name="testGetResponseWithInvalidToken" data-provider="InvalidTokens" finished-at="2022-03-04T21:11:08 IST" duration-ms="305" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[c9c1347c-698a-ef93-9504-43726dwf70c4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetResponseWithInvalidToken -->
        <test-method signature="testGetResponseWithInvalidToken(java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:08 IST" name="testGetResponseWithInvalidToken" data-provider="InvalidTokens" finished-at="2022-03-04T21:11:09 IST" duration-ms="305" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[c9c1647c-698a-gh93-9504-43726tyf70c4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetResponseWithInvalidToken -->
        <test-method signature="testGetResponseWithInvalidToken(java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:09 IST" name="testGetResponseWithInvalidToken" data-provider="InvalidTokens" finished-at="2022-03-04T21:11:09 IST" duration-ms="306" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[c9c1177c-698a-ij93-9504-43726zuf70c4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetResponseWithInvalidToken -->
        <test-method signature="testGetUsersPageInvalidToken(java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:09 IST" name="testGetUsersPageInvalidToken" data-provider="InvalidTokens" finished-at="2022-03-04T21:11:09 IST" duration-ms="312" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[c9c1347c-698a-ab93-9504-43726nhf70c4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetUsersPageInvalidToken -->
        <test-method signature="testGetUsersPageInvalidToken(java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:09 IST" name="testGetUsersPageInvalidToken" data-provider="InvalidTokens" finished-at="2022-03-04T21:11:10 IST" duration-ms="298" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[c9c1247c-698a-cd93-9504-43726adf70c4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetUsersPageInvalidToken -->
        <test-method signature="testGetUsersPageInvalidToken(java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:10 IST" name="testGetUsersPageInvalidToken" data-provider="InvalidTokens" finished-at="2022-03-04T21:11:10 IST" duration-ms="199" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[c9c1347c-698a-ef93-9504-43726dwf70c4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetUsersPageInvalidToken -->
        <test-method signature="testGetUsersPageInvalidToken(java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:10 IST" name="testGetUsersPageInvalidToken" data-provider="InvalidTokens" finished-at="2022-03-04T21:11:10 IST" duration-ms="239" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[c9c1647c-698a-gh93-9504-43726tyf70c4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetUsersPageInvalidToken -->
        <test-method signature="testGetUsersPageInvalidToken(java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:10 IST" name="testGetUsersPageInvalidToken" data-provider="InvalidTokens" finished-at="2022-03-04T21:11:10 IST" duration-ms="277" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[c9c1177c-698a-ij93-9504-43726zuf70c4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetUsersPageInvalidToken -->
        <test-method signature="testGetUsersPageWithoutToken()[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:10 IST" name="testGetUsersPageWithoutToken" finished-at="2022-03-04T21:11:12 IST" duration-ms="1942" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetUsersPageWithoutToken -->
        <test-method signature="testLogInForUnregisteredUsers(java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:12 IST" name="testLogInForUnregisteredUsers" data-provider="InvalidUserNamesAndPasswords" finished-at="2022-03-04T21:11:13 IST" duration-ms="764" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[FakeEmail1@fake.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1234567]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogInForUnregisteredUsers -->
        <test-method signature="testLogInForUnregisteredUsers(java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:13 IST" name="testLogInForUnregisteredUsers" data-provider="InvalidUserNamesAndPasswords" finished-at="2022-03-04T21:11:13 IST" duration-ms="204" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[FakeEmail2@fake.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1234567]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogInForUnregisteredUsers -->
        <test-method signature="testLogInForUnregisteredUsers(java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:13 IST" name="testLogInForUnregisteredUsers" data-provider="InvalidUserNamesAndPasswords" finished-at="2022-03-04T21:11:13 IST" duration-ms="207" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[FakeEmail3@fake.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1234567]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogInForUnregisteredUsers -->
        <test-method signature="testLogInForUnregisteredUsers(java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:13 IST" name="testLogInForUnregisteredUsers" data-provider="InvalidUserNamesAndPasswords" finished-at="2022-03-04T21:11:14 IST" duration-ms="211" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[FakeEmail4@fake.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1234567]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogInForUnregisteredUsers -->
        <test-method signature="testLogInForUnregisteredUsers(java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:14 IST" name="testLogInForUnregisteredUsers" data-provider="InvalidUserNamesAndPasswords" finished-at="2022-03-04T21:11:14 IST" duration-ms="211" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[FakeEmail5@fake.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1234567]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogInForUnregisteredUsers -->
        <test-method signature="testRegistration(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:14 IST" name="testRegistration" data-provider="NewData" finished-at="2022-03-04T21:11:14 IST" duration-ms="250" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[name1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Gszxyah11111@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [success] but found [The email address you have entered is already registered]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [success] but found [The email address you have entered is already registered]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.epam.TestAutomation.API_Assignment.TestAPI.testRegistration(TestAPI.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRegistration -->
        <test-method signature="testRegistration(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:14 IST" name="testRegistration" data-provider="NewData" finished-at="2022-03-04T21:11:14 IST" duration-ms="309" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[name2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Gszxyah21111@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [success] but found [The email address you have entered is already registered]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [success] but found [The email address you have entered is already registered]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.epam.TestAutomation.API_Assignment.TestAPI.testRegistration(TestAPI.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRegistration -->
        <test-method signature="testRegistration(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:14 IST" name="testRegistration" data-provider="NewData" finished-at="2022-03-04T21:11:15 IST" duration-ms="304" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[name3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Gszxyah31111@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123457]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [success] but found [The email address you have entered is already registered]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [success] but found [The email address you have entered is already registered]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.epam.TestAutomation.API_Assignment.TestAPI.testRegistration(TestAPI.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRegistration -->
        <test-method signature="testRegistration(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:15 IST" name="testRegistration" data-provider="NewData" finished-at="2022-03-04T21:11:15 IST" duration-ms="200" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[name4]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Gszxyah41111@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [success] but found [The email address you have entered is already registered]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [success] but found [The email address you have entered is already registered]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.epam.TestAutomation.API_Assignment.TestAPI.testRegistration(TestAPI.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRegistration -->
        <test-method signature="testRegistration(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:15 IST" name="testRegistration" data-provider="NewData" finished-at="2022-03-04T21:11:15 IST" duration-ms="304" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[name5]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Gszxyah51111@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [success] but found [The email address you have entered is already registered]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [success] but found [The email address you have entered is already registered]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.epam.TestAutomation.API_Assignment.TestAPI.testRegistration(TestAPI.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRegistration -->
        <test-method signature="testLogInWithValidUserNameAndWrongPassword(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:15 IST" name="testLogInWithValidUserNameAndWrongPassword" data-provider="NewData" finished-at="2022-03-04T21:11:16 IST" duration-ms="310" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[name1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Gszxyah11111@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogInWithValidUserNameAndWrongPassword -->
        <test-method signature="testLogInWithValidUserNameAndWrongPassword(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:16 IST" name="testLogInWithValidUserNameAndWrongPassword" data-provider="NewData" finished-at="2022-03-04T21:11:16 IST" duration-ms="309" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[name2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Gszxyah21111@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogInWithValidUserNameAndWrongPassword -->
        <test-method signature="testLogInWithValidUserNameAndWrongPassword(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:16 IST" name="testLogInWithValidUserNameAndWrongPassword" data-provider="NewData" finished-at="2022-03-04T21:11:16 IST" duration-ms="212" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[name3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Gszxyah31111@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123457]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogInWithValidUserNameAndWrongPassword -->
        <test-method signature="testLogInWithValidUserNameAndWrongPassword(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:16 IST" name="testLogInWithValidUserNameAndWrongPassword" data-provider="NewData" finished-at="2022-03-04T21:11:16 IST" duration-ms="199" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[name4]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Gszxyah41111@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogInWithValidUserNameAndWrongPassword -->
        <test-method signature="testLogInWithValidUserNameAndWrongPassword(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:16 IST" name="testLogInWithValidUserNameAndWrongPassword" data-provider="NewData" finished-at="2022-03-04T21:11:17 IST" duration-ms="297" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[name5]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Gszxyah51111@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogInWithValidUserNameAndWrongPassword -->
        <test-method signature="testLogIn(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:17 IST" name="testLogIn" finished-at="2022-03-04T21:11:17 IST" depends-on-methods="com.epam.TestAutomation.API_Assignment.TestAPI.testRegistration" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.epam.TestAutomation.API_Assignment.TestAPI.testLogIn() on instance com.epam.TestAutomation.API_Assignment.TestAPI@6392827e depends on not successfully finished methods 
[com.epam.TestAutomation.API_Assignment.TestAPI.testRegistration() on instance com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.epam.TestAutomation.API_Assignment.TestAPI.testLogIn() on instance com.epam.TestAutomation.API_Assignment.TestAPI@6392827e depends on not successfully finished methods 
[com.epam.TestAutomation.API_Assignment.TestAPI.testRegistration() on instance com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:149)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogIn -->
        <test-method signature="testGetUserDetailsRequestWithUserId(int,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:17 IST" name="testGetUserDetailsRequestWithUserId" finished-at="2022-03-04T21:11:17 IST" depends-on-methods="com.epam.TestAutomation.API_Assignment.TestAPI.testLogIn" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.epam.TestAutomation.API_Assignment.TestAPI.testGetUserDetailsRequestWithUserId() on instance com.epam.TestAutomation.API_Assignment.TestAPI@6392827e depends on not successfully finished methods 
[com.epam.TestAutomation.API_Assignment.TestAPI.testLogIn() on instance com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.epam.TestAutomation.API_Assignment.TestAPI.testGetUserDetailsRequestWithUserId() on instance com.epam.TestAutomation.API_Assignment.TestAPI@6392827e depends on not successfully finished methods 
[com.epam.TestAutomation.API_Assignment.TestAPI.testLogIn() on instance com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:149)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetUserDetailsRequestWithUserId -->
        <test-method signature="testGetUsersRequest(int,java.lang.String)[pri:0, instance:com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]" started-at="2022-03-04T21:11:17 IST" name="testGetUsersRequest" finished-at="2022-03-04T21:11:17 IST" depends-on-methods="com.epam.TestAutomation.API_Assignment.TestAPI.testLogIn" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.epam.TestAutomation.API_Assignment.TestAPI.testGetUsersRequest() on instance com.epam.TestAutomation.API_Assignment.TestAPI@6392827e depends on not successfully finished methods 
[com.epam.TestAutomation.API_Assignment.TestAPI.testLogIn() on instance com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.epam.TestAutomation.API_Assignment.TestAPI.testGetUsersRequest() on instance com.epam.TestAutomation.API_Assignment.TestAPI@6392827e depends on not successfully finished methods 
[com.epam.TestAutomation.API_Assignment.TestAPI.testLogIn() on instance com.epam.TestAutomation.API_Assignment.TestAPI@6392827e]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:149)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetUsersRequest -->
      </class> <!-- com.epam.TestAutomation.API_Assignment.TestAPI -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
